metainfo.patch

```diff
@@ -0,0 +1,17 @@
+--- a/src/Doom.metainfo.xml.in
++++ b/src/Doom.metainfo.xml.in
+@@ -44,4 +44,14 @@
+       <caption>Doom the Way id Did, on Freedoom. Level 3-2: City of Corpses</caption>
+     </screenshot>
+   </screenshots>
++  <content_rating type="oars-1.1">
```

Comment:
Should be contributed upstream if possible

Reply:
I agree.  I am intending to put in an upstream PR once I have a final, working, and approved version here.

Reply:
Thanks!